src/components/dropdown/revo-dropdown.style.scss

```
@@ -194,10 +219,17 @@ revo-dropdown {
-    background-color: #fff;
+    background-color: var(--rv-dropdown-bg, #fff);
+    border-color: var(--rv-dropdown-border, rgb(0 0 0 / 8%));
+    color: var(--rv-dropdown-color, rgba(0, 0, 0, 0.87));
+    box-shadow: 0 18px 40px rgba(15, 23, 42, 0.18);
 
     .filter-box {
       border: 1px solid $border-color;
+      background: var(--rv-dropdown-input-bg, #fff);
+      border-color: var(--rv-dropdown-border, $border-color);
+      color: var(--rv-dropdown-color, rgba(0, 0, 0, 0.87));
+      -webkit-text-fill-color: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)